manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dir": "ltr",
+  "lang": "en",
+  "name": "Cognizance",
+  "display": "fullscreen",
+  "start_url": "/",
```

Comment:
The start_url should be "http://polla.bhstudents.net/cognizance/", right?  It looks like "/" is short for "http://localhost/".

Reply:
I was thought this made it a relative path.

I did not want to hard code to https://wiki-cdsm.netdirect.ca/display/CDW/Patient+Treatment+Plan